modules/virt-about-virtualization-overview.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assembly:
+//
+// * virt/logging_events_monitoring/virt-reviewing-vm-dashboard.adoc
+//
+
+[id="virt-about-reviewing-top-consumers_{context}"]
+= About reviewing top consumers
```

Comment:
Again, this is more of a reference module and should likely be renamed.

Reply:
Please see my response in a previous comment section.

Reply:
Agree. Changed it to a reference module.

Reply:
This module is no longer used. It was replaced by a module with a different title.